devito/finite_differences/utils.py

```diff
@@ -0,0 +1,84 @@
+from functools import partial
+
+from devito.finite_differences.finite_difference import left, right, centered
+from devito.finite_differences.derivative import Diff
+
+def partial_derivative(expr, deriv_order, dims, fd_order, stagger=centered, side=centered):
+    new_obj = Diff(expr, dims, deriv_order=deriv_order, fd_order=fd_order, stagger=centered, side=centered)
```

Comment:
maybe 

`new_obj = Diff(expr, deriv_order=deriv_order, dims, fd_order=fd_order, stagger=centered, side=centered)`

just to keep the pattern ?